timemachine/cpp/src/math_utils.cuh

```diff
@@ -1,9 +1,11 @@
 #pragma once
 
-__device__ __host__ double sign(double a) { return (a > 0) - (a < 0); }
+int __forceinline__ ceil_divide(int x, int y) { return (x + y - 1) / y; }
 
-__device__ __host__ float sign(float a) { return (a > 0) - (a < 0); }
+// __device__ __host__ double sign(double a) { return (a > 0) - (a < 0); }
 
-__device__ __host__ double sign(Surreal<double> a) { return (a.real > 0) - (a.real < 0); }
+// __device__ __host__ float sign(float a) { return (a > 0) - (a < 0); }
 
-__device__ __host__ float sign(Surreal<float> a) { return (a.real > 0) - (a.real < 0); }
+// __device__ __host__ double sign(Surreal<double> a) { return (a.real > 0) - (a.real < 0); }
+
+// __device__ __host__ float sign(Surreal<float> a) { return (a.real > 0) - (a.real < 0); }
```

Comment:
was commenting these out intentional? Would it make sense to just remove them if unused?